### PR TITLE
Hilmar refactor Gelato Omen integration

### PR DESCRIPTION
**Pull request by hilmarx**

### CODE REVIEW

File: app/src/services/cpk.ts [hunk 1]
```diff
@@ -670,14 +672,20 @@ class CPKService {
...
       collateralAmount,
       collateralAddress,
       collateralDecimals,
+      networkId,
     )
 
+    // @dev Add the required amount of collateral the User needs
     if (!meetsGelatoThreshold) {
-      throw Error('belowGelatoThreshold')
+      throw Error('To activate an auto-withdraw, you need to at least: {X Token_Name}')
```
Comment (hilmarx) [inline app/src/services/cpk.ts:688]: See here: https://github.com/gelatodigital/gelato-omen/issues/45

File: app/src/common/constants.ts [hunk 2]
```diff
@@ -69,6 +69,8 @@ export const DEFAULT_ARBITRATOR: KnownArbitrator = 'kleros'
 
 export const DEFAULT_GELATO_CONDITION: KnownGelatoCondition = 'time'
```
Comment (superarius) [inline app/src/common/constants.ts:71]: yes to moving this, but reminder to us both this needs to be reset to 500 before PR

File: app/src/services/cpk.ts (hunk 1 above)
Comment (superarius) [reply app/src/services/cpk.ts:688]: yeah I'm into that idea, but just a note we also have to change [here](https://github.com/gelatodigital/omen-exchange/pull/1/files#diff-c7b1aa4985a3cb21d51e2c6e62e388efa9bebfbdf7542c720e6f0ffe9b073196R319-R323)

File: app/src/common/constants.ts (hunk 2 above)
Comment (hilmarx) [reply app/src/common/constants.ts:71]: Yes

**Review by superarius — APPROVED** [review]
This is great, thanks

**Pull request merged**